pkg/controller/storagecluster/reconcile.go

```diff
@@ -917,11 +917,16 @@ func newStorageClassDeviceSets(sc *ocsv1.StorageCluster) []rook.StorageClassDevi
 	return storageClassDeviceSets
 }
 
-func (r *ReconcileStorageCluster) throttleStorageDevices(storageClassName string) (bool, error) {
+func (r *ReconcileStorageCluster) throttleStorageDevices(ds ocsv1.StorageDeviceSet) (bool, error) {
```

Comment:
imho, it is a bit strange to have this check in the `throttleStorageDevices` function. It should be a different function or directly implemented in `ensureCephCluster`!?